### PR TITLE
Add --application-identifier for job_script create

### PR DESCRIPTION
**Pull request by zsinx6**
This PR is an update to the #69 to be easier to review

### CODE REVIEW

File: jobbergate_cli/jobbergate_api_wrapper.py [hunk 1]
```diff
@@ -455,42 +455,64 @@ def list_job_scripts(self, all):
...
+            debug                   --  optional parameter to view job script data
+                                        in CLI output
         """
-        if application_id is None:
+        parameter_check = []
+        if application_id and application_identifier:
             response = self.error_handle(
-                error="--application-id for the job script not defined",
-                solution="Please try again with --application-id specified",
+                error="both identifier and id supplied",
```
Comment (heitorPB) [inline jobbergate_cli/jobbergate_api_wrapper.py:479]:
```suggestion
                error="Both identifier and id supplied",
```

File: jobbergate_cli/jobbergate_api_wrapper.py [hunk 2]
```diff
@@ -455,42 +455,64 @@ def list_job_scripts(self, all):
...
-                                if this is not provided, the question askin in
-                                jobbergate.py is triggered
-            sbatch-params   --  optional parameter to submit raw sbatch parameters
-            fast            --  optional parameter to use default answers (when available)
-                                instead of asking user
-            debug           --  optional parameter to view job script data
-                                in CLI output
+            name                    --  Name for job script
+            application-id          --  id of the application for the job script
+            application-identifier  --  identifier of the application for the job script
```
[the remark this answers is not in the document]
Comment (zsinx6) [reply jobbergate_cli/jobbergate_api_wrapper.py:466]: The identifier is the new field we are adding, it is a unique human readable string

**Review by heitorPB — APPROVED** [review]
LGTM

**Comment by jamesbeedy** [comment]
👍

**Pull request merged**